bundles/org.openhab.binding.mideaac/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,394 @@
+			<channel id="outdoor-temperature" typeId="outdoor-temperature"/>
+			<channel id="sleep-function" typeId="sleep-function"/>
+			<channel id="temperature-unit" typeId="temperature-unit"/>
+
+			<channel id="on-timer" typeId="on-timer"/>
+			<channel id="off-timer" typeId="off-timer"/>
+			<channel id="appliance-error" typeId="appliance-error"/>
+			<channel id="auxiliary-heat" typeId="auxiliary-heat"/>
+			<channel id="maximum-humidity" typeId="maximum-humidity"/>
+			<channel id="humidity" typeId="humidity"/>
+			<channel id="kilowatt-hours" typeId="kilowatt-hours"/>
+			<channel id="amperes" typeId="amperes"/>
+			<channel id="watts" typeId="watts"/>
+			<channel id="screen-display" typeId="screen-display"/>
+			<channel id="filter-status" typeId="filter-status"/>
+		</channels>
+
+		<representation-property>ipAddress</representation-property>
+
+		<config-description>
+			<parameter name="ipAddress" type="text" required="true">
+				<context>ipAddress</context>
+				<label>IP Address</label>
+				<description>IP Address of the device.</description>
+			</parameter>
+			<parameter name="ipPort" type="decimal" required="true">
+				<context>ipPort</context>
+				<label>IP Port</label>
+				<description>IP port of the device.</description>
+				<default>6444</default>
+			</parameter>
+			<parameter name="deviceId" type="text" required="true">
+				<context>deviceId</context>
+				<label>Device ID</label>
+				<description>ID of the device. Leave 0 to do ID discovery.</description>
+				<default>0</default>
+			</parameter>
+			<parameter name="cloud" type="text" required="false">
+				<context>cloud</context>
+				<label>Cloud Provider</label>
+				<description>Cloud Provider name for email and password.</description>
+				<options>
+					<option value=""></option>
+					<option value="MSmartHome">MSmartHome</option>
+					<option value="Midea Air">Midea Air</option>
+					<option value="NetHome Plus">NetHome Plus</option>
+				</options>
+				<limitToOptions>true</limitToOptions>
+				<default>NetHome Plus</default>
+			</parameter>
+			<parameter name="email" type="text" required="false">
+				<context>email</context>
+				<label>Email</label>
+				<description>Email for cloud account chosen in Cloud Provider.</description>
+				<default>nethome+us@mailinator.com</default>
+			</parameter>
+			<parameter name="password" type="text" required="false">
+				<context>password</context>
+				<label>Password</label>
+				<description>Password for cloud account chosen in Cloud Provider.</description>
+				<default>password1</default>
+			</parameter>
+			<parameter name="token" type="text" required="false">
+				<context>token</context>
+				<label>Token</label>
+				<description>Secret Token (length 128 HEX) used for secure connection authentication used with devices v3 (if not
+					known, enter email, password for Cloud account to retrieve it).</description>
+			</parameter>
+			<parameter name="key" type="text" required="false">
```

Comment:
I think we can drop this as it is retrieved by the binding, there is no need to make this a configuration option.